docker/quickstart/docker-compose.yml

```diff
@@ -39,13 +42,42 @@ services:
       KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
       KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://localhost:9092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://${HOSTNAME}:9092
```

Comment:
${HOSTNAME} ?

Reply:
Yes, otherwise I wasn't able to access the kafka machine from outside :(

Reply:
Hmmm. Is `HOSTNAME` supported by non-unix systems?

Reply:
QQ: Are you deploying Kafka into a different host?

Reply:
Not right now. And not sure if HOSTNAME is supported by non-unix systems.  I will revert this property for now, and leave it in our version.

Reply:
Sure, let's leave it unchanged for now. We only tested running all Docker containers on a single host for now. Please contribute back how to best deploy containers to different hosts (maybe a Kubernetes solution ?) and you might want to refer to this article:
https://www.confluent.io/blog/kafka-listeners-explained/